_episodes/08-crop-raster-data.md

```diff
@@ -183,96 +174,41 @@ raster_clip.plot.imshow(figsize=(8,8))
-> ~~~
-> raster_clip_polygon2 = true_color_image.rio.clip([polygon_geom], true_color_image.rio.crs)
-> raster_clip_polygon2.plot.imshow()
-> ~~~
-> {: .language-python}
-> And try to compare the two methods:
-> - Do you have the same results?
-> - Do you have the same execution time?
-> - How would you choose the two methods in your daily work?
->
-> > ## Solution
-> >
-> > The two methods give the same results, but cropping directly with a polygon is much slower.
-> >
-> > Therefore, if the AoI is much smaller than the original raster, it would be more efficient to first crop the raster with a bounding box, then crop with the actual AoI polygon.
-> {: .solution}
-{: .challenge}
-
-> ## Exercise: Select the raster data within crop fields
-> Imagine that we want to select out all the crop fields from the raster data,
-> using the `.shp` file in `data/crop_fields`.
->
-> 1. Convert the coordinate system of `raster_clip` to the coordinate
-system of `cf_boundary_crop`. Tip: we can use `rio.reproject()` function.
-> 2. Crop the `raster_clip`.
-> 3. Visualize the cropped data.
-> 4. Store the raster as a GeoTIFF file (`crop_fields.tif`). Tip: we learned how to do this in the first episode.
->
-> > ## Solution
-> >
-> > ~~~
-> > # Load the crop fields polygons
-> > cf_boundary_crop = gpd.read_file("data/crop_fields/cf_boundary_crop.shp")
-> >
-> > # Convert the coordinate system
-> > raster_clip = raster_clip.rio.reproject(cf_boundary_crop.crs)
-> >
-> > # Crop
-> > raster_clip_fields = raster_clip.rio.clip(cf_boundary_crop['geometry'], cf_boundary_crop.crs)
-> >
-> > # Visualize
-> > raster_clip_fields.plot.imshow(figsize=(8,8))
-> >
-> > # Save
-> > raster_clip_fields.rio.to_raster("crop_fields.tif", driver="COG")
-> > ~~~
-> > {: .language-python}
-> > <img src="../fig/20-crop-raster-crop-fields-solution-06.png" title="Raster cropped by crop fields" width="512" style="display: block; margin: auto;" />
-> {: .solution}
-{: .challenge}
-
 
 ## Crop raster data with a geometry buffer
 
-It is not always the case that the AoI comes in the format of a polygon. Sometimes one would like to perform analysis around a (set of) point(s), or polyline(s). For example, in our AoI, there are also some groundwater monitoring wells coming as point vector data. One may also want to perform analysis around these wells. The location of the wells is stored in `data/groundwater_monitoring_well`.
+It is not always the case that the AoI comes in the format of polygon. Sometimes one would like to perform analysis around a (set of) point(s), or polyline(s). For example, in our AoI, there are also some groundwater monitoring wells coming as point vector data. One may also want to perform analysis around these wells. The location of the wells is stored in `data/groundwater_monitoring_well`. 
```

Comment:
AoI is not defined anywhere in this lesson. Maybe we want to mention what it is the first time we use the acronym.

Reply:
I think it was mentioned in Line 18 of this episode

> It is quite common that the raster data you have in hand is too large to process, or not all the pixels are relevant to your area of interest (AoI).

Reply:
Oops my bad! That's fine then!